2024-03-13 20:16:36,877 - regression_class_crypto_logistic - INFO - Running FSS
2024-03-13 20:16:38,108 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:16:38,108 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:16:38,117 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:16:38,117 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:16:38,121 - regression_class_crypto_logistic - INFO - Model 1
2024-03-13 20:16:38,192 - regression_class_crypto_logistic - INFO - Model 2
2024-03-13 20:16:38,325 - regression_class_crypto_logistic - INFO - Model 3
2024-03-13 20:16:38,435 - regression_class_crypto_logistic - INFO - Model 4
2024-03-13 20:16:38,556 - regression_class_crypto_logistic - INFO - Model 5
2024-03-13 20:16:38,764 - regression_class_crypto_logistic - INFO - Model 6
2024-03-13 20:16:38,923 - regression_class_crypto_logistic - INFO - Model 7
2024-03-13 20:16:39,110 - regression_class_crypto_logistic - INFO - Model 8
2024-03-13 20:25:38,539 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:25:38,547 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:25:38,549 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:25:38,550 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

